sdk/dotnet/GkeHub/FeatureMembership.cs

```diff
@@ -108,26 +108,26 @@ public sealed class FeatureMembershipArgs : Pulumi.ResourceArgs
         /// <summary>
         /// Config Management-specific spec.
         /// </summary>
-        [Input("configmanagement")]
-        public Input<Inputs.FeatureMembershipConfigmanagementArgs>? Configmanagement { get; set; }
+        [Input("configmanagement", required: true)]
```

Comment:
Breaking change.  Need to ensure that this is handled in accordance with our breaking changes policy.

Reply:
Same comment applies to all newly required fields in this file.